content/en/docs/concepts/architecture/cloud-controller.md

```diff
@@ -1,162 +1,144 @@
-
-### 2. Kubelet
-
-The Node controller contains the cloud-dependent functionality of the kubelet. Prior to the introduction of the CCM, the kubelet was responsible for initializing a node with cloud-specific details such as IP addresses, region/zone labels and instance type information. The introduction of the CCM has moved this initialization operation from the kubelet into the CCM.
-
-In this new model, the kubelet initializes a node without cloud-specific information. However, it adds a taint to the newly created node that makes the node unschedulable until the CCM initializes the node with cloud-specific information. It then removes this taint.
+### Route controller
 
-## Plugin mechanism
+The route controller is responsible for configuring routes in the cloud
+appropriately so that containers on different nodes in your Kubernetes
+cluster can communicate with each other.
 
-The cloud controller manager uses Go interfaces to allow implementations from any cloud to be plugged in. Specifically, it uses the CloudProvider Interface defined [here](https://github.com/kubernetes/cloud-provider/blob/9b77dc1c384685cb732b3025ed5689dd597a5971/cloud.go#L42-L62).
+Depending on the cloud provider, the route controller might also allocate blocks
+of IP addresses for the Pod network.
 
-The implementation of the four shared controllers highlighted above, and some scaffolding along with the shared cloudprovider interface, will stay in the Kubernetes core. Implementations specific to cloud providers will be built outside of the core and implement interfaces defined in the core.
+### Service controller
 
-For more information about developing plugins, see [Developing Cloud Controller Manager](/docs/tasks/administer-cluster/developing-cloud-controller-manager/).
+{{< glossary_tooltip text="Services" term_id="service" >}} integrate with cloud
+infrastructure components such as managed load balancers, IP addresses, network
+packet filtering, and target health checking. The service controller interacts with your
+cloud provider's APIs to set up load balancers and other infrastructure components
+when you declare a Service resource that requires them.
 
 ## Authorization
 
-This section breaks down the access required on various API objects by the CCM to perform its operations.
+This section breaks down the access that the cloud controller managers requires
+on various API objects, in order to perform its operations.
 
-### Node Controller
+### Node controller {#authorization-node-controller}
 
-The Node controller only works with Node objects. It requires full access to get, list, create, update, patch, watch, and delete Node objects.
+The Node controller only works with Node objects. It requires full access
+to GET, LIST, CREATE, UPDATE, PATCH, WATCH, and DELETE Node objects.
 
-v1/Node:
+`v1/Node`:
 
-- Get
-- List
-- Create
-- Update
-- Patch
-- Watch
-- Delete
+- GET
+- LIST
+- CREATE
+- UPDATE
+- PATCH
+- WATCH
+- DELETE
```

Comment:
I think this change is incorrect. By convention, when we use capital verbs, we mean verb used in the HTTP requests. These verbs were not capitalized because we wanted to differentiate them from HTTP verbs. For example, there is no HTTP request called 'LIST' or 'WATCH'.

Reply:
Pls refer to https://kubernetes.io/docs/reference/access-authn-authz/authorization/#determine-the-request-verb

Reply:
@sftim, What do you think about leaving the parameter names as in the original text?
Or is there another reason to make this change?

Reply:
Makes sense